apps/hubble-stats/components/Header/Header.tsx

```diff
@@ -0,0 +1,36 @@
+import { BlockIcon, GridFillIcon } from '@webb-tools/icons';
+import {
+  Breadcrumbs,
+  BreadcrumbsItem,
+  Chip,
+} from '@webb-tools/webb-ui-components';
+import Link from 'next/link';
+
+export const Header = () => {
+  return (
+    <div className="flex items-center justify-between pt-6 pb-4">
```

Comment:
Consider create a breadcrumb props for this Header component to use different breadcrumb for each page

Reply:
Instead of a `breadcrumbs` prop, I believe it would be better if we created a hook named `useBreadcrumbs()`. This hook would return a list of values for the `breadcrumbs` component. We could encapsulate all the breadcrumbs logic and calculations inside this hook. What do you think, @devpavan04 and @vutuanlinh2k2?

Reply:
Yes this sounds good to me. I can create and improve upon hubble-stats breadcrumbs as we start adding new routes.